src/lib/defaultCommands.js

```diff
@@ -596,6 +652,18 @@ define(function (require) {
 				});
 
 				editor.createDropDown(caller, 'insertlink', content);
+
+				// Check it is an email address.
+				if ($anchor.attr('href') &&
```

Comment:
Could you assign `$anchor.attr('href')` to a variable as that should help with minification. Variables can be minified more where as multiple `.attr('href')` can't be and should be less work for the browser.